bin/nodenv-default-packages

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+#
+# Summary: Refresh default packages on all installed node versions
+#
+# Usage: nodenv default-packages install [ --all | <version>...]
+#
+# Install default packages on a specific version or all installed versions.
+
+set -e
+[ -n "$NODENV_DEBUG" ] && set -x
+
+versions() {
+  case "$1" in
+    "")    nodenv version-name    ;; # list active version
+    --all) nodenv versions --bare ;; # list all installed versions
```

Comment:
We should also add `--skip-aliases` here (for `nodenv versions`). This will skip symlinks since running the install in a symlinked version would be duplicating and install that will have already occurred (or will occur) for the specific version itself. (ie, a symlink of `8` to `8.0.0` would cause the packages to be reinstalled _twice_ in the 8.0.0 node.)